### PR TITLE
Sidewalk

### PR DESCRIPTION
**Pull request by benjello**

### CODE REVIEW

File: liam2/alignment.py [hunk 1]
```diff
@@ -340,6 +340,7 @@ def compute(self, context, score, need=None,
             # calculate without leave_filter and without take_filter
             if need is None:
                 need = sum(score)
+            need = np.floor(need).astype(int)
```
Comment (gdementen) [inline liam2/alignment.py]: Wouldn't it be better to handle this as another option for frac_need? (e.g. "ignore" or "floor") and possibly use that option by default for sidewalk, if it makes more sense (which I am not convinced of).

Comment (benjello) [inline liam2/alignment.py]: I can only deal with this set up as far as i understand the implementation right now.
@AlexisEidelman any comments ?

**Comment by gdementen** [comment]
@benjello Hi, could you provide me with an example of values where it fails without the change? Thanks.

**Comment by gdementen** [comment]
@benjello hello? I will eventually close this PR without merging it if I don't have a test case for this.

**Comment by gdementen** [comment]
@benjello ping?

**Comment by benjello** [comment]
I think I do not use it anymore.Le us close and I will reopen if needed.

**Pull request closed without merging**